src/App.js

```diff
@@ -15,20 +16,21 @@ class App extends Component {
       lat: '',
       lon: '',
       name: '',
+      error: '',
     }
   }
   getLocation = async () => {
     const API = `https://us1.locationiq.com/v1/search.php?key=${cityKey}&q=${this.state.searchQuery}&format=json`;
     try{
       const response = await axios.get(API);
-      console.log(response.data);
+      // console.log(response.data);
       this.setState({location: response.data[0]});
       this.setState({lat: response.data[0].lat});
       this.setState({lon: response.data[0].lon});
       this.setState({name: response.data[0].display_name});
       this.setState({image: `https://maps.locationiq.com/v3/staticmap?key=${cityKey}&center=${this.state.lat},${this.state.lon}&zoom=12`});
```

Comment:
Nice work on using try catch blocks!


This can all be done in one setState() call here instead of using many setState calls.